webpack.config.js

```diff
@@ -19,8 +19,11 @@ module.exports = async (env, options) => {
     devtool: "source-map",
     entry: {
       polyfill: ["core-js/stable", "regenerator-runtime/runtime"],
-      vendor: ["react", "react-dom", "core-js", "@fluentui/react-components", "@fluentui/react-icons"],
-      taskpane: ["./src/taskpane/index.jsx", "./src/taskpane/taskpane.html"],
+      react: ["react", "react-dom"],
+      taskpane: {
+        import: ["./src/taskpane/index.jsx", "./src/taskpane/taskpane.html"],
+        dependOn: "react",
+      },
       commands: "./src/commands/commands.js",
```

Comment:
So, as I understand it, we're not including FluentUI in any of the bundles anymore. Is that right?

Reply:
I made the fix to get the bundle size down and in doing so was able to remove the fluentui components here.  I believe the still get included in the taskpane bundle file because that is where the components get imported and so webpack includes them there.